pkg/api/types.go

```diff
@@ -1205,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
+	// Fs represents a filesystem-based local volume
+	// +optional
+	Fs *LocalFsVolume
+}
+
+// LocalFsVolume represents a filesystem-based local volume
+type LocalFsVolume struct {
```

Comment:
The idea here is to leave room for block devices, yeah?  What about all the other volume types that didn't do this - can we find a way to solve it for everyone instead of being new for this one?